docs/dev/setup/localci-localvc.rst

```diff
@@ -143,11 +143,78 @@ Setup with Docker Compose
 
 You can also use Docker Compose to set up the local CI and local VC systems. Using the following command, you can start the Artemis and MySQL containers:
 
-::
+.. code-block:: bash
 
     docker compose -f docker/artemis-dev-local-vc-local-ci-mysql.yml up
 
 .. HINT::
     Unix systems: When running the Artemis container on a Unix system, you will have to give the user running the container permission to access the Docker socket by adding them to the ``docker`` group. You can do this by changing the value of ``services.artemis-app.group_add`` in the ``docker/artemis-dev-local-vc-local-ci-mysql.yml`` file to the group ID of the ``docker`` group on your system. You can find the group ID by running ``getent group docker | cut -d: -f3``. The default value is ``999``.
 
     Windows: If you want to run the Docker containers locally on Windows, you will have to change the value for the Docker connection URI. You can add ``ARTEMIS_CONTINUOUSINTEGRATION_DOCKERCONNECTIONURI="tcp://host.docker.internal:2375"`` to the environment file, found in ``docker/artemis/config/dev-local-vc-local-ci.env``. This overwrites the default value ``unix:///var/run/docker.sock`` for this property defined in ``src/main/resources/config/application-docker.yml``.
+
+
+Podman as Docker alternative
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`Podman <https://podman.io/>`_ offers a container runtime that is API-compatible with Docker.
+Rather than having a system-wide socket that runs with administrative permissions, Podman allows to create containers with only user permissions.
+In single-user setups this might not be as relevant, but offers additional security in a production environment where the Artemis CI has to execute untrusted student code.
+
+.. admonition:: Podman is supported on a best effort basis.
+
+    We are relying on the API compatibility to provide support but are not actively testing against Podman on a test system or in the continuous integration.
```

Comment:
It's correct grammar to leave out `the` here, as it does not refer to exactly one specific CI. (Grammarly detected this, too)